src/vos/vos_pool.c

```diff
@@ -1272,6 +1272,26 @@ vos_pool_create_ex(const char *path, uuid_t uuid, daos_size_t scm_sz, daos_size_
 	return rc;
 }
 
+int
+vos_pool_roundup_size(daos_size_t *scm_sz, daos_size_t *meta_sz)
+{
+	int    backend;
+	size_t alignsz;
+
+	backend = umempobj_get_backend_type();
+	if ((*scm_sz != *meta_sz) && (backend == DAOS_MD_BMEM))
+		backend = DAOS_MD_BMEM_V2;
```

Comment:
i think we should return failure in this case?

Reply:
Why, this seems valid to me. It is the standard case for phase 2 where mem_ratio is not 100% and therefore we need to use V2.

Reply:
i thought that if users configured backend as DAOS_MD_BMEM using environment DAOS_MD_ON_SSD_MODE. I would think we don't support phase2 in this case? As phase1 and Phase2 will be totally different layout.

Reply:
Addressed in the latest commit. Also, added a new tunable DAOS_MD_DISABLE_BMEM_V2 to disable creation of v2 pools.